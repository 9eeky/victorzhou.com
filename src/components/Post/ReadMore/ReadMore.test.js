import React from 'react';
import renderer from 'react-test-renderer';
import ReadMore from './ReadMore';

const post = () => ({
  frontmatter: {},
});

describe('ReadMore', () => {
  it('renders correctly', () => {
    const props = {
      nextPost: post(),
      prevPost: post(),
    };

    const tree = renderer.create(<ReadMore {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
