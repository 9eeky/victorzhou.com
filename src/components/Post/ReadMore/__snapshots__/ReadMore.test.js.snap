// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReadMore renders correctly 1`] = `
<div
  className="readmore"
>
  <h4
    className="readmore-title"
  >
    READ MORE
  </h4>
  <div
    className="pagination readmore-pagination"
  >
    <div
      className="pagination__prev"
    >
      <Link
        className="pagination__prev-link"
        rel="prev"
        to="/"
      >
        ← PREV
      </Link>
    </div>
    <div
      className="pagination__next"
    >
      <Link
        className="pagination__next-link"
        rel="next"
        to="/"
      >
        → NEXT
      </Link>
    </div>
  </div>
</div>
`;
